src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -78,17 +116,24 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 			}
 		}
 
-		let tooltip = 'Server: ' + connectionProfile.serverName + '\r\n' +
-			'Database: ' + (connectionProfile.databaseName ? connectionProfile.databaseName : '<default>') + '\r\n';
+		let tooltip = localize('status.connection.baseTooltip', 'Server: {0}\nDatabase: {1}\n', connectionProfile.serverName,
```

Comment:
nit: Remove the last `\n` here and on line 123 and just add it when you append the new content to the tooltip. Right now this will often result in unnecessary newlines at the end of the tooltips